packages/fxa-auth-server/lib/metrics/glean/server_events.ts

```diff
@@ -85,8 +79,6 @@ class AccountsEventsServerEvent {
     utm_source,
     utm_term,
   }: {
-    user_agent: string;
-    ip_address: string;
```

Comment:
@akkomar some backend work hasn't been merged back...?  I want to be consistent with the version of glean.js in the monorepo but can back out of these changes since the issue was on accounts_frontend.

Reply:
@chenba looks like these fields landed in glean_parser v9 and glean.js is using v8.1.1. I suggest to back out the server changes from this PR.
I will follow up with Bruno about bumping parser version in glean.js.